region/brinstar/pink/Spore Spawn Super Room.json

```diff
@@ -209,6 +209,87 @@
         "This is possible with an IBJ, but it's still about 3 minutes, and you can't remotely acquire the item because it's in a Chozo Ball."
       ]
     },
+    {
+      "link": [2, 1],
+      "name": "G-Mode Overload PLMS with Camera Scroll Blocks, Shinespark",
```

Comment:
To damage down after the CF to trigger reserves when at high ETank count
- Shinesparking is 145 damage and quick to repeat.  But doing a shinespark over and over should require some shortcharge lenience.
- Standing on the bug spawner felt like vaguely 30s per etank.  And there is an implied farm after this strat before continuing to the next room.
This makes this series of strats slower than they appear.